src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -31,6 +31,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String username = request.getParameter("username");
     String password = request.getParameter("password");
+    String about = request.getParameter("about");
+    Boolean isAdmin = false;
+
+    if (request.getParameter("admin password") == "123") {
```

Comment:
I'd make the parameter admin_password. Properties like this usually don't include spaces.